apps/src/templates/BaseDialog.jsx

```diff
@@ -31,7 +31,8 @@ export default class BaseDialog extends React.Component {
     bodyClassName: PropTypes.string,
     style: PropTypes.object,
     soundPlayer: PropTypes.object,
-    overflow: PropTypes.string
+    overflow: PropTypes.string,
+    backdropExtension: PropTypes.number
```

Comment:
this is too specific of a use-case for it to be a prop. we could make this more flexible by having the prop be `backdropStyle` (same as the `bodyStyle` prop that exists in this component) and adding a comment that the prop is temporary and thus shouldn't be depended on